livvkit/components/validation.py

```diff
@@ -117,9 +117,11 @@ def run_suite(case, config):
     summary = _summarize_result(m, result)
     _print_summary(m, case, summary)
 
+
     functions.create_page_from_template("validation.html",
                                         os.path.join(livvkit.index_dir, "validation", case + ".html"))
-    functions.write_json(result, os.path.join(livvkit.output_dir, "validation"), case + ".json")
+    with open(os.path.join(livvkit.output_dir, "validation", case + ".json"), 'w') as f:
+        f.write(result._repr_json())
```

Comment:
Ooh, good catch -- you may want to just remove `functions.write_json` as well. Looks like it's only used here, and in `test_functions.py`

Reply:
Thanks Joe! I think I'll leave it for now, since the test function uses it to test `functions.read_json`